java/src/main/java/com/powsybl/dataframe/network/extensions/BranchObservabilityDataframeProvider.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.dataframe.network.extensions;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.dataframe.network.NetworkDataframeMapper;
+import com.powsybl.dataframe.network.NetworkDataframeMapperBuilder;
+import com.powsybl.dataframe.network.adders.NetworkElementAdder;
+import com.powsybl.iidm.network.Branch;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.extensions.BranchObservability;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+/**
+ * @author Etienne Lesot <etienne.lesot@rte-france.com>
+ */
+@AutoService(NetworkExtensionDataframeProvider.class)
+public class BranchObservabilityDataframeProvider implements NetworkExtensionDataframeProvider {
+
+    private Stream<BranchObservability> itemsStream(Network network) {
+        return network.getBranchStream().filter(Objects::nonNull)
+                .map(branch -> (BranchObservability) branch.getExtension(BranchObservability.class))
+                .filter(Objects::nonNull);
+    }
+
+    private BranchObservability getOrThrow(Network network, String id) {
+        Branch branch = network.getBranch(id);
+        if (branch == null) {
+            throw new PowsyblException("Invalid branch id : could not find " + id);
+        }
+        return (BranchObservability) branch.getExtension(BranchObservability.class);
+    }
+
+    @Override
+    public String getExtensionName() {
+        return BranchObservability.NAME;
+    }
+
+    @Override
+    public NetworkDataframeMapper createMapper() {
+        return NetworkDataframeMapperBuilder.ofStream(this::itemsStream, this::getOrThrow)
+                .stringsIndex("id", branchObservability -> ((Branch) branchObservability.getExtendable()).getId())
+                .booleans("observable", BranchObservability::isObservable)
+                .doubles("p1_standard_deviation", branchObservability -> branchObservability.getQualityP1().getStandardDeviation(),
```

Comment:
As identified on another evolution:
we must take care of the cases where quality will be null.

In that case we can return false/nan for redundant/standard dev.